content/en/_index.md

```diff
@@ -299,6 +392,14 @@ try {
   // IllegalArgumentException's are thrown when an invalid argument has been passed to a function. Ok to ignore.
 }
 {{< /tab >}}
+{{< tab header="Ruby" >}}
+    # Follow the Google Ruby style guide and catch the most specific type of Exception, instead of a more general one.
```

Comment:
Does Google have a Ruby style guide?

If not I think it's fine to update this to just say "catch the most specific type of Exception, instead of a more general one."

Reply:
yes we have a style guide for Ruby https://engdoc.corp.google.com/eng/doc/rubyguide.html?cl=head

Reply:
Is there a public version? It came up in the node.js PR that if we reference something we should be using a link, and we ended up just removing the reference to the java style guide (leaving it what I suggested above).

Reply:
Sure will update this for Ruby too